src/olympia/reviewers/tests/test_views.py

```diff
@@ -5184,14 +5184,14 @@ def test_reused_guid_from_previous_deleted_addon(self):
         assert response.status_code == 200
         assert b'Previously deleted entries' not in response.content
 
-        old_one = addon_factory(status=amo.STATUS_DELETED)
-        old_two = addon_factory(status=amo.STATUS_DELETED)
-        old_other = addon_factory(status=amo.STATUS_DELETED)
-        old_noguid = addon_factory(status=amo.STATUS_DELETED)
-        ReusedGUID.objects.create(addon=old_one, guid='reuse@')
-        ReusedGUID.objects.create(addon=old_two, guid='reuse@')
-        ReusedGUID.objects.create(addon=old_other, guid='other@')
-        ReusedGUID.objects.create(addon=old_noguid, guid='')
+        old_one = addon_factory(status=amo.STATUS_DELETED, guid=None)
```

Comment:
Why is it necessary to set `guid` to `None` now, when it wasn't before? Is this just making the test more explicit?

Reply:
It's to match the states the addon instances would be in reality.  If they're supposed to be deleted and re-uploaded then the guid would have been reset on the old instance after the new upload.

(If we don't set the guid to `None` then we'd need to find and update the AddonGUID instance for them to point to `guid='reuse@'` instead - it'd be `AddonGUID.objects.get(addon=old_one).update(guid='reuse@')`)

Reply:
... and the guid hasn't been set to `None` since 2016, when _someone_ changed it in #1471.

Reply:
Now it's back to not setting `guid` to `None`.